### PR TITLE
Improve code for flip and card detail

### PR DESCRIPTION
**Pull request by veredrec**
Relates #79

* Clean code on `card detail`- passed `selected` to render function and remove this.props
* Fix bug with flip - now when the card is flipped and user clicks `like` or `dislike` card flips back to normal

### CODE REVIEW

File: src/components/career_detail.jsx [hunk 1]
```diff
@@ -9,26 +8,26 @@ class CareerDetail extends Component {
 
     this.state = {};
   }
-  render() {
+  render({ selected }) {
```
Comment (Karyum) [inline src/components/career_detail.jsx]: huh nice didn't even know you could do that

**Comment by Karyum** [comment]
@veredrec i didn't merge anything how all of a sudden you got a merge conflict?

**Comment by veredrec** [comment]
@Karyum I have no idea, but I'll solve the conflict and push again, thanks!

**Comment by veredrec** [comment]
@Karyum I solved it :smile:

**Pull request merged**